src/Error/CoercionError.php

```diff
@@ -0,0 +1,40 @@
+<?php declare(strict_types=1);
+
+namespace GraphQL\Error;
+
+use GraphQL\Language\AST\Node;
+use GraphQL\Utils\Value;
+use Throwable;
+
+/**
+ * @phpstan-type Path list<string>
```

Comment:
The path for values in an input list contains `int`. Also, let's differentiate this from the Path used for fields - how about `InputPath`? Same for the name of the property.

Reply:
ValuePath? Since its not only for input but also for enum && scalar.

Reply:
Returned results are also values, but GraphQL distinguishes between input and output - see https://spec.graphql.org/draft/#sec-Input-and-Output-Types

Reply:
Renamed to InputPath